calico_node/startup/startup.go

```diff
@@ -236,7 +239,7 @@ func getNode(client *client.Client, nodeName string) *api.Node {
 
 // configureIPsAndSubnets updates the supplied node resource with IP and Subnet
 // information to use for BGP.
-func configureIPsAndSubnets(node *api.Node) {
+func configureIPsAndSubnets(node *api.Node) bool {
```

Comment:
One final thing:  I think the comment should indicate that it returns true if the node IP addresses are updated.